tests/unit/test_schema.py

```diff
@@ -20,7 +20,10 @@
 
 from google.cloud import bigquery
 from google.cloud.bigquery.standard_sql import StandardSqlStructType
-from google.cloud.bigquery.schema import PolicyTagList
+from google.cloud.bigquery.schema import (
+    PolicyTagList,
+    SerDeInfo,
+)
```

Comment:
Same nit here, regarding https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Hmm. I attempted to correct this for both `PolicyTagList` and `SerDeInfo`.
Unfortunately, there is something about how PolicyTagList is handled in our code that did not like the change and it produced several failing tests. I did not want to clutter this PR with unrelated changes, nor did I want to delay the issuance of this PR.

So I chose to do the following:
* I corrected `SerDeInfo` and throughout the code we reference `schema.SerDeInfo`.
* I left the PolicyTagList code exactly as it was found.